pkg/arvo/sys/hoon.hoon

```diff
@@ -5940,7 +5940,7 @@
     (rush txt ;~(pfix ;~(plug sig dot) urs:ab))
   ::
       %tas
-    (rush txt sym)
+    (rush txt ;~(pose (jest '') sym))
```

Comment:
Only changing this here creates a mismatch between `+slaw` and `+slay`.

The comment in `+de-beam` (from @Fang-?) suggests updating `+sym` with something like `;~(pose low (easy %$))`. But I'm not sure if that's too dramatic a change.

Reply:
The comment is yours. (;

That would certainly be a very dramatic change, `sym` is used in parsers all over the place and has so far generally been understood to mean "an actual symbol". Allowing the empty string for the empty symbol would be a very breaking change. Adding a `+sim` might be preferable.

Reply:
lol